decompiler/VuDisasm/VuDisassembler.cpp

```diff
@@ -411,15 +412,15 @@ VuProgram VuDisassembler::disassemble(void* data, int size_bytes, bool debug_pri
 
     // debug
     if (debug_print) {
-      fmt::print("{}\n", to_string(VuInstructionPair{upper_instr, lower_instr}));
+      lg::debug("{}\n", to_string(VuInstructionPair{upper_instr, lower_instr}));
```

Comment:
can we leave this file as normal prints too? The output is stuff that I copy and paste directly into .cpp files.